aitelemetry/telemetrywrapper.go

```diff
@@ -90,6 +92,14 @@ func NewAITelemetry(
 	id string,
 	aiConfig AIConfig,
 ) TelemetryHandle {
+	env, _ := os.LookupEnv("AZACN_TESTENV")
+	if env != "test" {
+		cloudName, err := common.GetAzureCloud()
+		if cloudName != azurePublicCloud {
+			debuglog("[AppInsights] This is not azure public cloud.%s err:%v", cloudName, err)
```

Comment:
debuglog - is this a std. convention? I have seen camel case used in general ( i.e. similar to debugLog )

Reply:
remove . after cloud = cloud.%s